src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -184,6 +189,29 @@ private String commandStringGenerator(
     return commandString.toString();
   }
 
+  /**
+   * Reads Serverconfig from File pathToResourceDocker/ssh.rc
+   */
+  private void readServerConfig() {
+    String configFilePath = pathToResourceDocker + "/ssh.rc";
+    try {
+      List<String> lines = Files.readAllLines(Paths.get(configFilePath), Charset.forName("UTF-8"));
+      for (String line : lines) {
+        if (line.contains("p2p_server=") && !line.endsWith("=")) {
```

Comment:
Nice method!
Two things I'm wondering, maybe I'm just lacking the knowledge of the code:
1. Why always "[1]"? Maybe I'm understanding it wrong, but this sounds like the line number in the list. So in this case it wouldn't make sense - in a normal "for"-loop there would be the variable "i", right?
2. What happens if the file is not in this particular form or somehow corrupt? e.g. typo or whatever... It would be able to read the file but not storing anything, but also without "saying" anything. My suggestion: using the logger to state what got importet.